packages/testing/src/renderWeb3Hook.tsx

```diff
@@ -36,11 +35,7 @@ export const renderWeb3Hook = async <Tprops, TResult>(
   const { result, waitForNextUpdate, rerender, unmount } = renderHook<Tprops, TResult>(hook, {
     wrapper: (wrapperProps) => (
       <MockWeb3Wrapper connector={connector}>
-        <BlockNumberProvider>
-          <ChainStateProvider multicallAddresses={multicallAddresses}>
-            <UserWrapper {...wrapperProps} />
-          </ChainStateProvider>
-        </BlockNumberProvider>
```

Comment:
We can't just remove core features to make something look better. Without that wrapper, we can't test any of the hooks calling the contracts.

Reply:
That's why i think tests should provide wrappers of providers, because some tests might need different wrappers